lib/file_parser/email_mime.py

```diff
@@ -114,6 +116,8 @@ def maintain_rfc_parse(message):
     :return: Returns a email message formatted as a string
       :rtype: str
     """
+    if message is None:
+        return "".as_string()
```

Comment:
I think this might not work

Reply:
Apologies. You're absolutely right. Fixed that with [69bc05f](https://github.com/seunomosowon/TA-mailclient/pull/25/commits/69bc05f4f03c279334c06aea1fca46ed484d9f96). 

Basically, if you have for some reason a raw email message beginning with `\n` and `maintain_rfc` parameter set to `True`, you'd end up having a `None` object passed to `maintain_rfc_parse()` and raising 
```bash
AttributeError: 'NoneType' object has no attribute 'is_multipart'
```

Change offered in [69bc05f](https://github.com/seunomosowon/TA-mailclient/pull/25/commits/69bc05f4f03c279334c06aea1fca46ed484d9f96) will help preventing this error to occur. See also [here](https://stackoverflow.com/questions/17873091/python-msg-email-message-from-stringaaa-values-are-returning-none).